docs/spec/v1beta1/kustomization.md

```diff
@@ -622,9 +623,35 @@ spec:
 
 The `targetNamespace` is expected to exist.
 
+### Patches
+
+To add [Kustomize `patches` entries](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/)
+to the configuration, and patch resources using either a [strategic merge](https://kubectl.docs.kubernetes.io/references/kustomize/glossary#patchstrategicmerge) 
+patch or a [JSON](https://kubectl.docs.kubernetes.io/references/kustomize/glossary#patchjson6902) patch,
+`spec.patches` items must contain a `target` selector and a `patch` document.
```

Comment:
`items must contain a target` but in the blow example there is no target specified

Reply:
Thank you for the catch :)

In the original Kustomize resource, it's possible to apply a strategic merge patch (not JSON6902) without specifying the target.
I decided to force the target to be present in the Flux implementation for simplicity. Maybe that is something could be added in the future.